query/src/main/scala/filodb/query/exec/MetadataExecPlan.scala

```diff
@@ -395,75 +348,96 @@ final case class LabelCardinalityExec(queryContext: QueryContext,
     s" startMs=$startMs, endMs=$endMs"
 }
 
-final case object TopkCardExec {
-  // TODO: tune this
-  // See here for help choosing a size:
-  //   https://datasketches.apache.org/docs/Frequency/FrequentItemsErrorTable.html
-  val MAX_ITEMSKETCH_MAP_SIZE = 256;
+/**
+ * Contains utilities for all TsCardinality materialize() derivatives.
+ */
+final case object TsCardExec {
+  val MAX_RESPONSE_SIZE = 5000  // TODO: tune this
```

Comment:
There are multiple constants NMAX, and this one. Can we consolidate ?

Reply:
`MAX_RESPONSE_SIZE` is network-limited, and (formerly-named) `NMAX` is memory-limited. If those two are similar enough, I'll just use `NMAX` in both places. Does that sound good?

Reply:
Ended up doing this, so marking resolved